shared/utils/generate-from-templates.py

```diff
@@ -20,14 +20,18 @@ def __init__(self):
             "file_dir_permissions.csv":     "create_permission.py",
             "accounts_password.csv":        "create_accounts_password.py",
         }
-        self.supported_ovals      = ["oval_5.10"]
-        self.langs                = ["bash", "ansible", "oval"]
-        utils_dir                 = os.path.dirname(os.path.realpath(__file__))
-        root_dir                  = os.path.join(utils_dir, "..", "..")
-        self.shared_templates_dir = os.path.join(root_dir, "shared", "templates")
+        self.supported_ovals = ["oval_5.10"]
+        self.langs = ["bash", "ansible", "oval"]
+        utils_dir = os.path.dirname(os.path.realpath(__file__))
+        root_dir = os.path.join(utils_dir, "..", "..")
+        self.shared_templates_dir = \
+            os.path.join(root_dir, "shared", "templates")
 
         self.current_oval = "oval_5.10"
 
+    def set_langs(self, langs):
```

Comment:
We could but the script and templating don't really care what the language is. The only real special case is `oval`.